docs/dictionary/keyword/bold.lcdoc

```diff
@@ -27,16 +27,13 @@ the object) or a chunk in a field or button.
 
 To add boldfacing to an object without removing other styles (such as
 italic), add the <bold> <keyword> to the end of the <textStyle>. The
-following example adds bold to the styles of a <button> :
+following example adds bold to the styles of a <button>:
 
     if the textStyle of button 1 is empty then
-
-    set the textStyle of button 1 to bold
+        set the textStyle of button 1 to bold
```

Comment:
`"bold"` should be quoted here as this will not compile in strict compilation mode